docs/contributing/making-changes-to-a-pull-request.md

```diff
@@ -0,0 +1,17 @@
+# Making changes to a Pull Request
+
+When a Pull Request branch is [checked out](review-a-pull-request-in-visual-studio.md) you can commit changes to it and push and pull like any other branch. If the Pull Request branch is located in a fork and was checked out from the GitHub Pull Request details pane, then a remote to that fork will be created automatically and the branch set to track the fork branch.
+
+## Pulling Changes
```

Comment:
> Pulling Changes

I think the concept of pulling is easier to understand if you expand on this, maybe:

> Pulling changes to your local clone